src/main/java/uk/gov/hmcts/reform/lrdapi/domain/BuildingLocationStatus.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.lrdapi.domain;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import java.io.Serializable;
+import java.util.Set;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+
+@Entity(name = "building_location_status")
+@NoArgsConstructor
+@EntityListeners(AuditingEntityListener.class)
+@Getter
```

Comment:
EntityListeners are we have any explicit requirement of listeners. not getting why are using EntityListeners can you please explain?

Reply:
Hi @abhijitdiwan I guess this was a copy and paste error. I will remove this.